src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Node/JsonNodeConverter.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Text.Json.Node;
+
+namespace System.Text.Json.Serialization.Converters
+{
+    internal class JsonNodeConverter : JsonConverter<object>
+    {
+        public static JsonNodeConverter Default { get; } = new JsonNodeConverter();
```

Comment:
nit: what does it mean to be the default in this case? Can there be a non-"default" instance?

```suggestion
        public static JsonNodeConverter Instance { get; } = new JsonNodeConverter();
```